packages/ember-glimmer/tests/integration/syntax/each-in-test.js

```diff
@@ -23,25 +26,32 @@ applyMixins(EachInTest,
 
   new TruthyGenerator([
     // TODO: figure out what the rest of the cases are
-    { foo: 1 }
+    { foo: 1 },
+    EmberObject.create({ 'Not Empty': 1 }),
+    ObjectProxy.create({ content: { 'Not empty': 1 } }),
+    ObjectProxy.create({ content: Object.create({}) }),
+    ObjectProxy.create({ content: EmberObject.create() })
   ]),
 
   new FalsyGenerator([
     // TODO: figure out what the rest of the cases are
     {},
     Object.create({ 'Not Empty': 1 }),
+    Object.create({}),
+    EmberObject.create(),
+    ObjectProxy.create({}),
+    // TODO: These 2 should be falsy but are returning true
+    //ObjectProxy.create({ content: null }),
+    //ObjectProxy.create({ content: {} }),
     undefined,
     null
   ])
-
-  // TODO(mmun): Add support for object proxies and
```

Comment:
The object test cases pass in objects for both truthy and falsy values: `{ isTruthy: this.falsyValue }`, so it will always render T1T2. If we want to include them here perhaps I should update https://github.com/emberjs/ember.js/pull/13441 to better reflect the test cases we care about rather than testing for objects with isTruthy.